cockroachcloud/cloud-sso.md

```diff
@@ -47,26 +47,26 @@ After Enterprise Authentication is enabled, your Cockroach Labs team or the supp
 
 ## {{ site.data.products.db }} SSO Frequently Asked Questions (FAQ)
 
-### Will it work if I try to sign in with SSO without explicitly setting it up for my account, if I already use an email from an SSO Provider (e.g., my-example-account@gmail.com)?
+### Will it work if I try to sign in with SSO without explicitly setting it up for my account, if I already use an email from an SSO Provider such as Gmail?
```

Comment:
Do we want SSO-related changes in this CMEK rotation-related PR?

Reply:
Yes because those changes missed the original SSO PR so I took care of them here.